applications/jonny5/src/j5_balance_crawler.erl

```diff
@@ -0,0 +1,229 @@
+%%%-------------------------------------------------------------------
+%%% File    : j5_balacne_crawler.erl
+%%% Description : Jonny5 module for disconnect calls when account
+%%% balance drops below zero
+%%%-------------------------------------------------------------------
+-module(j5_balance_crawler).
+
+-behaviour(gen_server).
+
+%% API
+-export([start_link/0]).
+
+%% gen_server callbacks
+-export([init/1
+         ,handle_call/3
+         ,handle_cast/2
+         ,handle_info/2
+         ,terminate/2
+         ,code_change/3
+        ]).
+
+-export([maybe_disconnect_account/1]).
+-export([delayed_hangup/2]).
+
+-include("jonny5.hrl").
+
+-record(state, {}).
+-define(SERVER, ?MODULE).
+-define(CRAWLER_CYCLE, 60000).
+-define(INTERACCOUNT_DELAY, 10).
+
+%%====================================================================
+%% API
+%%====================================================================
+%%--------------------------------------------------------------------
+%% Function: start_link() -> {ok,Pid} | ignore | {error,Error}
+%% Description: Starts the server
+%%--------------------------------------------------------------------
+start_link() ->
+  gen_server:start_link(?MODULE, [], []).
+
+%%====================================================================
+%% gen_server callbacks
+%%====================================================================
+
+%%--------------------------------------------------------------------
+%% Function: init(Args) -> {ok, State} |
+%%                         {ok, State, Timeout} |
+%%                         ignore               |
+%%                         {stop, Reason}
+%% Description: Initiates the server
+%%--------------------------------------------------------------------
+init([]) ->
+    wh_util:put_callid(?MODULE),
+    self() ! 'crawl_accounts',
```

Comment:
I would use `gen_server:cast/2` only because this is a known message and `handle_info` is typically for out-of-band/unexpected message handling.